.nycrc

```diff
@@ -10,7 +10,6 @@
     "**/docs",
     "**/samples",
     "**/scripts",
-    "**/src/**/v*/**/*.js",
```

Comment:
Here is a change from synthtool templates:
https://github.com/googleapis/synthtool/commit/af05219a18fb36da8efa96fb0d816c004ad1e71d